cclunit-framework-source/doc/examples/mocking_api.inc

```diff
@@ -0,0 +1,70 @@
+;;; put the following script definition in a .prg file and compile it
+
+
+drop program mock_other_script go
+create program mock_other_script
```

Comment:
Maybe document this as
/**
Mock implementation for other_script which sets reply equal to a copy of mock_other_script_reply.
*/

Reply:
Will do.